.github/workflows/RELEASE.yaml

```diff
@@ -101,6 +89,16 @@ jobs:
           git config user.name "github-actions[bot]"
           git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
 
+      - name: Check if new version implies pre-release
+        id: check_prerelease
+        run: |
+          if [[ ${{ github.event.inputs.version }} =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            PRE_RELEASE=false
+          else
+            PRE_RELEASE=true
+          fi
+          echo "isPreRelease=${PRE_RELEASE}" >> $GITHUB_OUTPUT
+
```

Comment:
I like that, we should maybe just use that in the SDK as well. Currently, we are explicitly asking for alpha release there. This here makes more sense 👍

Reply:
Created https://github.com/camunda/connector-sdk/issues/305

Reply:
Thank you!